pkg/curatedpackages/reader.go

```diff
@@ -12,6 +12,10 @@ import (
 	releasev1 "github.com/aws/eks-anywhere/release/api/v1alpha1"
 )
 
+const (
+	defaultRegistry = "783794618700.dkr.ecr.us-west-2.amazonaws.com"
```

Comment:
Why is this a constant here? What account is this?

Reply:
This is our private prod account. We need this constant because currently packages are not included in the release manifest and we need a way to figure out the registry to pull from.